storage-image-classification/extension.yaml

```diff
@@ -0,0 +1,87 @@
+name: storage-label-image
+version: 0.0.1
+specVersion: v1beta
+
+displayName: Image Labelling
+description: Performs image labelling on images uploaded to Storage and writes labels to storage.
+
+license: Apache-2.0  # The license you want for the extension
+
```

Comment:
reminder to add `sourceUrl`. example:

https://github.com/FirebaseExtended/experimental-extensions/blob/d65bfa7268fe6811f5fd1037b5127d76d6b0ecfa/firestore-auth-claims/extension.yaml#L26-L27